### PR TITLE
feat(mnist-summary): Add jupyter demo

### PR DESCRIPTION
**Pull request by gaocegege**

### CODE REVIEW

File: mnist/tensorflow/mnist_with_summaries/README.md [hunk 1]
```diff
@@ -4,9 +4,39 @@
 
 Install skai dashboard can refer this [doc](https://github.com/skai-x/dashboard/tree/master/charts/skai-dashboard).
 
+## Develop
```
Comment (xieydd) [inline mnist/tensorflow/mnist_with_summaries/README.md:7]: We need doc for install elastic jupyter in preparation, WDYT.

File: mnist/tensorflow/mnist_with_summaries/README.md [hunk 2]
```diff
@@ -4,9 +4,39 @@
...
+    metadata:
+      labels:
+        notebook: simple
+    spec:
+      containers:
+        - name: notebook
+          image: jupyter/base-notebook:python-3.9.7
+          command: ["tini", "-g", "--", "start-notebook.sh"]
+EOF
+
```
Comment (xieydd) [inline mnist/tensorflow/mnist_with_summaries/README.md:32]: Can we post the mnist jupyter file in this dir for user.

**Review by xieydd — CHANGES_REQUESTED** [review]

File: mnist/tensorflow/mnist_with_summaries/README.md (hunk 1 above)
Comment (gaocegege) [reply mnist/tensorflow/mnist_with_summaries/README.md:7]: SGTM, should we have the complete doc or drop a link to elastic jupyter operator doc here?

Comment (xieydd) [reply mnist/tensorflow/mnist_with_summaries/README.md:7]: I think link is ok for this doc.

**Comment by gaocegege** [comment]
The comments are addressed, PTAL @xieydd

**Comment by xieydd** [comment]
LGTM

**Review by xieydd — APPROVED** [review]

**Pull request merged**